src/features/Ad/index.tsx

```diff
@@ -2,6 +2,11 @@ import '../Ad/index.css';
 import CheckMark from '../../components/CheckMark';
 
 const Ad = (): JSX.Element => {
+  const handleSelect = (packageName: string, cost: string): void => {
+    const url = `/payment?package=${packageName}&cost=${cost}`;
+    window.location.href = url;
+  };
```

Comment:
we are using a library for that `react-router-dom`. Out of my head, I think you should use the redirect() to do that

Reply:
Since it's last minute, I guess it will be refactor for next sprint